hazelcast/src/main/java/com/hazelcast/instance/HazelcastInstanceManager.java

```diff
@@ -47,13 +50,26 @@
 @SuppressWarnings("SynchronizationOnStaticField")
 @PrivateApi
 public final class HazelcastInstanceManager {
```

Comment:
Could you allign this with the HazelcastClientManager so that either all methods are static, or a singleton is used and the methods are not static? Now we have 1 class with static methods, and another with instance methods.